epoch/api/channels_api_usage.md

```diff
@@ -518,3 +521,432 @@ details:
+The `channel_id` with a combination with the correct singatures verifies that
+this off-chain transaction indeed is part of the channel off-chain state. The
+`round` represents the height of the channel's state at the time the
+transaction was co-signed by the parties. The higher the round - the newer the
+transaction is. This `round` must be greater than the last on-chain one for that channel.
+The `state_hash` is the internal channel state tree root hash
+at that `round` height.
+
+If the transaction's `payload` is empty - then the latest on-chain state for
+this channel is used. Both `channel_deposit` and `channel_withdraw`
+transactions contain a `round` and a `state_hash` and the latest received one
+overwrites the previous one. If there had been none of those, then the
+`channel_create` transaction is used: it has a `state_hash` and an implicit
+`round = 1`.
+
+Either by having a value in the `payload` or not having one, the
+`channel_solo_close` and `channel_slash` provide a channnel's `round` and a `state_hash`.
+In order to determine the order of the channel's states received - we compare
+the `rounds` and keep the state with the greatest `round`, considered to be
+the _newest_ and _latest_ state. They also provide the `state_hash` the
+channel's state tree had at this `round`.
+
+Both `channel_solo_close` and `channel_slash` contain a `poi` field. This is
+the proof of inclusion for participants' balances in the channel state: all
+the insignificant data in the channel's MPT is replaced by corresponding hashes.
+The root hash of the PoI must be equal to the `state_hash` provided by the `payload`.
+This guarantees that the PoI indeed is a proof of inclusion for tree at this height.
+
+#### Solo close on-chain transaction
+The `channel_close_solo` transaction is the one that triggers the solo closing
+sequence. After it is included on-chain channel enters a _closing_ state and
+any subsequent withdrawal or deposits are considered invalid. Preconditions for
+the `channel_close_solo` to be valid are:
+
+* channel is opened on-chain
+* channel is not in a _closing_ state but not yet closed - no `channel_close_solo` had been
+  included in a block yet
+
+Any participant in the channel can post a `channel_close_solo` transaction. In
+the scope of this description we will call the one that posts the transaction
+_solo closer_.
+The transaction has the following structure:
+
+  | Name | Type | Description |
+  | ---- | ---- | ----------- |
+  | channel id | string | ID of the channel to close |
+  | from | string | solo closer's public key |
+  | payload | binary | closing payload |
+  | poi | binary | closing proof of inclusion |
+  | ttl | integer | maximum block height to include the transaction |
+  | fee | integer | fee to be paid to the miner |
+  | nonce | integer | solo closer's nonce |
+
+`payload` and `poi` are validated as [described above](#payload-and-proof-of-inclusion)
```

Comment:
Is it really described above?

Reply:
I think it is: the values of PoI, payload and their connection with `state_hash` and `round`